lib/matplotlib/dviread.py

```diff
@@ -267,6 +267,8 @@ def _output(self):
             maxx = max(maxx, x + w)
             maxy = max(maxy, y + e)
             maxy_pure = max(maxy_pure, y)
+        if self._baseline_v is not None:
+            maxy_pure = self._baseline_v  # This should normally be the case.
```

Comment:
do we want to reset `self._baseline_v = None` here?

Reply:
sounds good